bazelrc/contrib.bazelrc

```diff
@@ -0,0 +1,47 @@
+# Copyright 2016 The Bazel Authors. All rights reserved.
```

Comment:
I'm not sure if we want this file?
What's the plan for the current bazelrc with Bazel versions and the latest.bazelrc?

Reply:
spoken offline, the references to latest.bazelrc for local docker sandbox or remote cache will be changed to point to this file. The latest.bazelrc files should be deprecated and even removed from our repo soon.